src/handler.rs

```diff
@@ -71,7 +78,44 @@ struct AddrMapping {
     gpa_base: u64,
 }
 
-pub struct VhostUserHandler<S, V, B: Bitmap + 'static> {
+#[derive(Default)]
+struct InflightFdState {
+    inflight_file: Option<File>,
+    inflight_mapping_addr: Option<GuestAddress>,
+    inflight_mmap_size: usize,
+}
+
+impl InflightFdState {
+    fn new() -> Self {
+        Self::default()
+    }
+
+    fn set_inflight_state(
+        &mut self,
+        inflight_file: Option<File>,
+        inflight_mapping_addr: Option<GuestAddress>,
+        inflight_mmap_size: usize,
+    ) {
+        self.inflight_file = inflight_file;
+        self.inflight_mapping_addr = inflight_mapping_addr;
+        self.inflight_mmap_size = inflight_mmap_size;
+    }
+
+    fn get_inflight_mapping_addr(&self) -> Option<GuestAddress> {
+        self.inflight_mapping_addr
+    }
+
+    fn get_inflight_mmap_size(&self) -> usize {
+        self.inflight_mmap_size
+    }
+}
+
+pub struct VhostUserHandler<S, V, B>
```

Comment:
Line 113 to 117 should have been changed to
```
pub struct VhostUserHandler<S, V, B: Bitmap + 'static> {
```
in last MR.  Seems something messed up during rebase?

Reply:
Yes, think it's fixed now.